src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -65,4 +85,16 @@ public function getBlockPrefix(): string
     {
         return 'country';
     }
+
+    private static function getEmojiFlag(string $name): string
```

Comment:
what about putting it on `Countries` in Intl? That would allow reusing for display purpose as well.

And then we can argue if the Form integration isnt just a custom label ...